spec/controllers/api/groups_controller_spec.rb

```diff
@@ -105,4 +105,14 @@
     end
   end
 
+  describe 'count_explore_results' do
+    it 'returns the number of explore group results matching the search term' do
+      group.update_attribute(:name, 'exploration team')
+      explore_group = FactoryGirl.create(:group, name: 'investigation team')
```

Comment:
No need for `FactoryGirl.create` here, just `create(:group, name: 'thing')` should work fine. 💛